FEEDBACK.md

```diff
@@ -0,0 +1,9 @@
+General comments/feedback:
+
+Overall think it looks great. Clear and easy to follow, and plenty of good content. Here's some general feedback. Let me know if any of these sound good and I can make separate PRs for them:
+
+1. Should we include a brief overview of programming in general at the very beginning? High level, ‘what is a program?’ sort of stuff?
```

Comment:
shit haha. ok maybe i'll just add it to the "logging " section then, or would that be too confusing...?